tests/Makefile.am

```diff
@@ -95,7 +95,7 @@ YAMI_VPP_CFLAGS = \
 
 yamidecode_LDADD    = $(YAMI_VPP_LIBS)
 yamidecode_LDFLAGS  = $(YAMI_VPP_LDFLAGS)
-yamidecode_SOURCES  = decode.cpp decodehelp.cpp $(DECODE_INPUT_SOURCES) decodeoutput.cpp vppinputoutput.cpp vppinputdecode.cpp vppoutputencode.cpp encodeinput.cpp encodeInputCamera.cpp encodeInputDecoder.cpp vppinputdecodecapi.cpp
+yamidecode_SOURCES  = decode.cpp decodehelp.cpp $(DECODE_INPUT_SOURCES) decodeoutput.cpp vppinputoutput.cpp vppinputdecode.cpp vppoutputencode.cpp encodeinput.cpp encodeInputCamera.cpp encodeInputDecoder.cpp vppinputdecodecapi.cpp md5.c
```

Comment:
We only want to compile md5.c if ENABLE_MD5 is defined from AM_CONDITIONAL in configure.ac

Reply:
this will introduce complicity, how about let md5 build by default. if we have libbsd we will use libbsd, if we do not have we fallback to a built in md5 sum function(slow one)